docs/user/installation_and_configuration.rst

```diff
@@ -408,14 +408,14 @@ your GeoServer ``WEB-INF/lib`` directory.
 * Zookeeper
     * zookeeper-3.4.6.jar
 
-GeoMesas GeoServer Community Module
------------------------------------
+GeoMesa's GeoServer Community Module
```

Comment:
Yeah, you could do "GeoMesa GeoServer Community Module" (or "The GeoMesa GeoServer Community Module") too. The way it was written it seemed like the author wanted to use a  possessive, which requires an apostrophe.